docs/sdk-upgrade.md

```diff
@@ -56,7 +56,6 @@ Steps:
     1. Look at the [Testing Considerations](#testing-considerations) for other ideas on what you might want to test.
     1. Here is a sample plan:
         - For each combination of devices and browsers above:
-            1. Set `doc_auth_selfie_capture_enabled: true` in your `application.yml`
```

Comment:
Confession: I removed this line, but am considering out of scope evaluating what this markdown file is and whether it's still relevant.